tests/prepare.ts

```diff
@@ -63,7 +76,7 @@ const log = async <T>(label: string, fn: () => Promise<T>) => {
           middleware: 'express',
           routesDir: './tests/fixtures/express',
         },
-        defaultOptions,
+        defaultOptions(2),
```

Comment:
Let's not call this over and over again. Assign the default options, assign `openApi3options` based on the `defaultOptions` and use these.